pkgs/applications/misc/feedbackd/default.nix

```
@@ -0,0 +1,51 @@
+{ stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook
+, glib
+, gsound
+, libgudev
+, json-glib
+, vala
+, gobject-introspection
+}:
+
+stdenv.mkDerivation rec {
+  pname = "feedbackd";
+  version = "0.0.0+git20201114";
```

```suggestion
  version = "2020-11-14";
```

as written here https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

---

i see that is their official release name. wtf. nevermind